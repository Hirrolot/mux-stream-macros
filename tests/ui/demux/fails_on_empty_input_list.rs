use mux_stream_macros::demux_with_error_handler;

fn main() {
    demux_with_error_handler!();
}
