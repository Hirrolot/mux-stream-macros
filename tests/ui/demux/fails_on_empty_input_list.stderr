error: At least one variant is required
 --> $DIR/fails_on_empty_input_list.rs:4:5
  |
4 |     demux_with_error_handler!();
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
